index.bs

```diff
@@ -249,20 +252,36 @@ and defensive programming which includes:
     enhanced by the possible usage of a sensor, not degraded by its
     absence.
 
-<div class="example">
+<div class="example" id="robust-example">
+    The following snippet illustrates how an Accelerometer sensor can be created
+    in a robust manner.
     <pre highlight="js">
-        try { // No need to feature detect thanks to try..catch block.
-            var sensor = new GeolocationSensor();
-            sensor.start();
-            sensor.onerror = error => gracefullyDegrade(error);
-            sensor.onreading = _ => updatePosition(sensor.latitude, sensor.longitude);
-        } catch(error) {
-            gracefullyDegrade(error);
+        let accelerometer = null;
+        try {
+            accelerometer = new Accelerometer({ frequency: 10 });
+            accelerometer.addEventListener('error', event => handleErrors(event.error));
```

Comment:
Maybe comment // handle runtime errors

Reply:
what happens when handleErrors throws from a runtime error?

Reply:
log to console

Reply:
Would be nice to add that as a comment above the throw

Reply:
Then the throw makes even less sense... why not print to console straight away in both cases, you could use console.error()

Reply:
Added comment for runtime errors.